fe/src/components/Login.vue

```diff
@@ -1,19 +1,20 @@
 <template>
   <div class="login">
     <el-row>
-      <el-col offset=9 span=6>
+      <el-col :offset="9" :span="6">
```

Comment:
what is the significance of these numbers

Reply:
Those numbers specify column width and offset width, that's how it's done in the element docs, let me know if you recommend doing the layout another way.